backend/src/app/rpc/mutations/teams.clj

```diff
@@ -359,12 +360,19 @@
               :role role))
       nil)))
 
+(def sql:upsert-team-invitation
+  "insert into team_invitation(team_id, email_to, role, valid_until)
+   values (?, ?, ?, ?)
+       on conflict(team_id, email_to) do
+          update set role = ?, valid_until = ?, updated_at = clock_timestamp();")
+
 (defn- create-team-invitation
   [{:keys [conn tokens team profile role email] :as cfg}]
   (let [member   (profile/retrieve-profile-data-by-email conn email)
+        token_exp (dt/in-future "48h")
```

Comment:
Prefer use `-` as separator instead of `_`

Reply:
fixed